frontend/src/views/Pages/ResetPassword.vue

```diff
@@ -92,7 +92,12 @@ export default {
         })
         .then(() => {
           this.form.password = ''
-          this.$router.push('/thx/reset')
+          if (this.$route.path.includes('checkEmail')) {
+            this.$router.push('/thx/checkEmail')
+          }
+          if (this.$route.path.includes('reset')) {
+            this.$router.push('/thx/reset')
+          }
```

Comment:
With this code it should check every if block, in my opinion the following would be a bit more efficient: 
```suggestion
          if (this.$route.path.includes('checkEmail')) {
            this.$router.push('/thx/checkEmail')
          } else if (this.$route.path.includes('reset')) {
            this.$router.push('/thx/reset')
          }
```